engine/runtime-compiler/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -186,13 +191,25 @@ class Compiler(
               shouldCompileDependencies
             )
 
+            if (generateDocs.isDefined) {
+              val v = if (generateDocs.get == "api") {
+                DocsVisit.createSignatures();
+              } else {
+                DocsVisit.createMarkdown();
+              }
```

Comment:
```suggestion
              val v = generateDocs.get == "api" ? DocsVisit.createSignatures() : DocsVisit.createMarkdown();
```
I'm a fan of ternary operator, yes :)

Reply:
I applied your suggestion, but the Scala compiler complains. There don't seem to be ternary operators in Scala...

Reply:
Back to if/then/else: 9bfd828

Reply:
Ah, it's Scala.... I got confused by all those semicolons...